packages/sdk-client-v3/src/types/types.d.ts

```diff
@@ -222,7 +222,7 @@ export type HttpMiddlewareOptions = {
   retryConfig?: RetryOptions
   httpClient: Function
   getAbortController?: () => AbortController
-  httpClientOptions?: object
+  httpClientOptions?: object // will be passed as a second argument to your httpClient function for configuration
```

Comment:
I noticed that from the docs link provided [here](https://github.com/commercetools/commercetools-docs/pull/4986) there is not mention of the `httpClientOptions` as an addition to the `httpMiddlewareOption` just as we have one for `httpClient`. Please add this property to the docs.

Reply:
I added it here: https://github.com/commercetools/commercetools-docs/pull/4986/files#diff-4c9187ddfa33195642947e7508f80e1fced9d8e01127e80d1c4b2d9544f46524R18

Reply:
I know it was added there, but I am saying is that it should have also been added [here](https://github.com/commercetools/commercetools-docs/pull/4986/files#diff-d58e725f4184915964670f40fe874774b1dbdfbb8e1a06419683540c7b1481d1R255) in the difference between the v2 and v3 middleware.

Reply:
Ah I see what you mean. I checked and it's not the right place to add it. The part where you pointed to is for migration from v2 to v3. Since this option was not available at all in v2, there is no need to migrate it.